onnxruntime/test/shared_lib/custom_op_utils.cc

```diff
@@ -43,7 +43,7 @@ void MyCustomKernel::Compute(OrtKernelContext* context) {
   void* allocated = allocator->Alloc(allocator, 2);
   EXPECT_NE(allocated, nullptr) << "KernelContext_GetAllocator() can successfully allocate some memory";
   allocator->Free(allocator, allocated);
-
+  ort_.ReleaseAllocator(allocator);
```

Comment:
> ort_.ReleaseAllocator(allocator); [](http://example.com/codeflow?start=1&length=34)

This is a memory leak again and not a good example. Let's wrap the pointer into Ort::Allocator so it is destroyed automatically.